R/find_formula.R

```diff
@@ -2,69 +2,67 @@
 #' @name find_formula
 #'
 #' @description Returns the formula(s) for the different parts of a model
-#'    (like fixed or random effects, zero-inflated component, ...).
-#'    `formula_ok()` checks if a model formula has valid syntax
-#'    regarding writing `TRUE` instead of `T` inside `poly()`
-#'    and that no data names are used (i.e. no `data$variable`, but rather
-#'    `variable`).
+#'  (like fixed or random effects, zero-inflated component, ...).
+#'  `formula_ok()` checks if a model formula has valid syntax
+#'  regarding writing `TRUE` instead of `T` inside `poly()`
+#'  and that no data names are used (i.e. no `data$variable`, but rather
+#'  `variable`).
 #'
 #' @param verbose Toggle warnings.
 #' @param ... Currently not used.
 #' @inheritParams find_predictors
 #'
 #' @return A list of formulas that describe the model. For simple models,
-#'    only one list-element, `conditional`, is returned. For more complex
-#'    models, the returned list may have following elements:
+#'  only one list-element, `conditional`, is returned. For more complex
+#'  models, the returned list may have following elements:
 #'
-#'    - `conditional`, the "fixed effects" part from the model (in the
-#'      context of fixed-effects or instrumental variable regression, also
-#'      called *regressors*) . One exception are `DirichletRegModel` models
-#'      from \pkg{DirichletReg}, which has two or three components,
-#'      depending on `model`.
+#'  - `conditional`, the "fixed effects" part from the model (in the
+#'    context of fixed-effects or instrumental variable regression, also
+#'    called *regressors*) . One exception are `DirichletRegModel` models
+#'    from **DirichletReg**, which has two or three components,
```

Comment:
What motivates this change? `\pkg{}` -> `**`?

I would prefer it we instead use `` `{pkg}` `` because then pkgdown can link to that package's website (if available). You don't need to revert this change. Just for future reference.

Reply:
`\pkg{}` was just rendered as bold text, that's why I replaced it with the related markdown tag.

Reply:
We could use regex to replace, shouldn't be too hard to check occurrences